test/built-ins/Temporal/Duration/compare/order-of-operations.js

```diff
@@ -134,6 +134,8 @@ const expectedOpsForPlainRelativeTo = expected.concat([
   "get options.relativeTo.year.valueOf",
   "call options.relativeTo.year.valueOf",
   "call options.relativeTo.calendar.dateFromFields",
+  // lookup
```

Comment:
Usually the lookup is in the algorithm steps for the method itself, rather than in an AO, so that's why it's not labeled with an AO. (I prefer not to put the actual step numbers unless it's really needed for clarity, since they bitrot so easily.) I can try to label these with things like `// lookup in Duration.compare` instead.